client/src/views/Settings/Tabs/ChangePassword/index.vue

```diff
@@ -70,12 +92,47 @@ export default {
         confirmNewPassword: ''
       }
     }
+  },
+
+  methods: {
+    requiredField,
+    checkPasswordField,
+    checkConfirmPassword (newval) {
+      return checkRepeatPasswordField(newval, this.formModel.newPassword)
+    },
+    onForgotPasswordClick () {
+      this.$router.push({ path: '/forgotpassword' })
+    },
+    onFormSubmit () {
+      authService.changePassword({
+        oldPassword: this.formModel.currentPassword,
+        newPassword: this.formModel.newPassword
+      })
+        .then((response) => {
+          this.$q.notify({
+            color: 'primary',
+            textColor: 'white',
+            message: 'Your password was changed.',
+            actions: [{ icon: 'close', color: 'white' }],
+            timeout: 3000
+          })
+        })
+        .catch(err => {
+          this.$q.notify({
+            color: 'negative',
+            textColor: 'white',
+            message: err.response.data,
```

Comment:
You need to check if a response in error exists, an empty error will throw reference error.
Additionally textColor, actions, and timeout sharable in $q.notify, you could move them as sharableObject.